copy.go

```diff
@@ -66,27 +71,8 @@ func doCopy(dst net.Conn, src net.Conn, buf []byte, writeTimeout time.Duration,
 	}
 }
 
-func writeTo(dst net.Conn, buf []byte, writeTimeout time.Duration) error {
```

Comment:
`writeTimeout` is not used in `BidiCopy` too. If we don't want to break the signature, maybe rename it as `unused`